vocabularies/Org.OData.Temporal.V1.xml

```diff
@@ -136,26 +147,20 @@
       <!-- Operations on Temporal Objects -->
 
       <Action Name="Update" IsBound="true" EntitySetPath="timeslices">
-        <Annotation Term="Core.Description" String="Updates all time slices of a temporal object in a given period of application time after splitting partially overlapping time slices" />
+        <Annotation Term="Core.Description" String="Updates existing time slices with values from new time slices where the latter overlap the former" />
+        <Annotation Term="Core.LongDescription">
+          <String>(Splitting of existing time slices will be explained here.)</String>
+        </Annotation>
         <Parameter Name="timeslices" Type="Collection(Edm.EntityType)" Nullable="false">
-          <Annotation Term="Core.Description" String="Time slices to modify" />
-        </Parameter>
-        <Parameter Name="From" Type="Edm.PrimitiveType" Nullable="false">
-          <Annotation Term="Core.Description" String="Lower boundary (inclusive) of application time. Date or DateTimeOffset value, or string with value `min`" />
-        </Parameter>
-        <Parameter Name="To" Type="Edm.PrimitiveType" Nullable="false">
-          <Annotation Term="Core.Description" String="Upper boundary (exclusive) of application time. Date or DateTimeOffset value, or string with value `max`" />
+          <Annotation Term="Core.Description" String="Collection of time slices to be updated" />
         </Parameter>
-        <Parameter Name="Values" Type="Edm.EntityType" Nullable="false">
-          <Annotation Term="Core.Description" String="Property values to update" />
-          <Annotation Term="Core.LongDescription" String="Same entity type as binding parameter and return type, specify only values of non-key properties." />
-        </Parameter>
-        <Parameter Name="ObjectKey" Type="Edm.EntityType" Nullable="false">
-          <Annotation Term="Core.OptionalParameter">
-            <Record />
+        <Parameter Name="newValues" Type="Collection(Edm.EntityType)" Nullable="false">
+          <Annotation Term="Core.Description" String="Partial time slices whose property values are used to update the `timeslices` collection" />
+          <Annotation Term="Core.LongDescription">
+            <String>Same entity type as binding parameter.
+            The partial time slices need not contain all properties, but at least the boundary values of the interval to change.

```

Comment:
If the time-slice collection has non-temporal key properties, but these are not given in `newValues`, all "temporal objects" in the collection will be updated at once. OK?

Reply:
Yes, similar to SQL's `UPDATE ... FOR PORTION OF BUSINESS_TIME ... WHERE ... `, see https://www.ibm.com/docs/en/db2oc?topic=statements-update